redis/redis.go

```diff
@@ -59,6 +60,29 @@ type store struct {
 	*redistore.RediStore
 }
 
+// GetRedisStore get the actual woking stiore.
+//
+// Ref: https://godoc.org/github.com/boj/redistore#RediStore
+func GetRedisStore(s Store) (err error, rediStore *redistore.RediStore) {
+	if realStore, ok := s.(*store); !ok {
+		err = errors.New("unable to get the redis store: Store isn't *store")
+	} else {
+		rediStore = realStore.RediStore
+	}
+
+	return
+}
+
+// SetKeyPrefix sets the key prefix in the redis database.
+func SetKeyPrefix(s Store, prefix string) error {
+	if err, rediStore := GetRedisStore(s); err != nil {
+		return err
+	} else {
```

Comment:
remove `else` block, just leave `rediStore.SetKeyPrefix(prefix)`

Reply:
This won't work as you've done because rediStore is in the scope of the if statement, I will make a PR